### DISCUSSION TITLE
why the hell is the .env not in the gitnore?

### DISCUSSION THREAD

**Discussion by ghost** [cboard-org/cboard#1788] [General]
why

**Comment by sylvansson** [comment]
Hi @BlazarKnight, as far as I can tell the .env file is already listed in .gitignore ([ref](https://github.com/cboard-org/cboard/blob/56b5f3419b6fb722d9fc84fab8467a6dfaced152/.gitignore#L39)).

**Reply by ghost** [reply]
then what is this???  https://github.com/cboard-org/cboard/tree/master/env

**Reply by ghost** [reply]
Why does it have private keys in it??????

**Reply by sylvansson** [reply]
According to the [README](https://github.com/cboard-org/cboard/blob/bedff3023b261df411923f2d0962b88763652152/README.md#secrets-management), prod-private.gpg contains secrets that were encrypted with a key that only @shayc and a few others have access to. When decrypted they're stored in a folder ignored by Git ([ref](https://github.com/cboard-org/cboard/blob/56b5f3419b6fb722d9fc84fab8467a6dfaced152/.gitignore#L4)).
